tests/algorithms/pytorch/test_newton_raphson.py

```diff
@@ -355,11 +354,13 @@ def test_pytorch_nr_algo_performance(
 
 @pytest.mark.slow
 @pytest.mark.substra
-def test_download_load_algo(network, compute_plan, session_dir, nr_test_data, mae, rtol):
-    download_algo_files(
-        client=network.clients[0], compute_plan_key=compute_plan.key, round_idx=None, dest_folder=session_dir
+def test_download_load_algo(network, compute_plan, nr_test_data, mae, rtol):
+    algo = download_algo_state(
+        client=network.clients[0],
+        compute_plan_key=compute_plan.key,
+        round_idx=NUM_ROUNDS,
```

Comment:
I might be missing something, but wouldn't the equivalent of the previous test be with `round_idx=None`?

Reply:
your not, I used it to test sorry :) I changed it back